api/src/main/java/org/apache/gravitino/credential/S3SecretKeyCredential.java

```diff
@@ -29,26 +29,26 @@ public class S3SecretKeyCredential implements Credential {
   /** S3 secret key credential type. */
   public static final String S3_SECRET_KEY_CREDENTIAL_TYPE = "s3-secret-key";
   /** The static access key ID used to access S3 data. */
-  public static final String GRAVITINO_S3_ACCESS_KEY_ID = "s3-access-key-id";
+  public static final String GRAVITINO_S3_STATIC_ACCESS_KEY_ID = "s3-static-access-key-id";
   /** The static secret access key used to access S3 data. */
-  public static final String GRAVITINO_S3_SECRET_ACCESS_KEY = "s3-secret-access-key";
+  public static final String GRAVITINO_S3_STATIC_SECRET_ACCESS_KEY = "s3-static-secret-access-key";
```

Comment:
I mean that we can still follow AWS convention to use "s3-secret-access-key", but the definition here in Gravitino can be a more differentiable name. like:

```java
public static final String GRAVITINO_S3_STATIC_SECRET_ACCESS_KEY = "s3-secret-access-key";
```
Also for other keys, what do you think? @FANNG1

Reply:
ok

Reply:
updated